src/main/java/org/truffleruby/language/SetTopLevelBindingNode.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved. This
```

Comment:
In Graal.js there is a mandatory comma after the year. Graal seems to use the same. I don't think this has any consequence, just noting the difference.